requirements.txt

```diff
@@ -25,4 +25,5 @@ bleach
 django-fullurl
 unidecode
 stop_words
-django-markdownify
\ No newline at end of file
+django-markdownify
```

Comment:
what's this used for?

Reply:
markdownify ? was an easy way to allow for simple formatting in public notes